usaspending_api/settings.py

```diff
@@ -122,6 +122,9 @@
 ES_REPOSITORY = ""
 ES_ROUTING_FIELD = "recipient_agg_key"
 
+# Grants API
+GRANTS_API_KEY = os.environ.get("GRANTS_API_KEY")
```

Comment:
Make sure you work with OPS on this, the older pattern was to not use env vars but inject the config into the settings,py file using regex. I never learned why....

Reply:
we can make this work. We will handle the same way as the database secrets. We should probalby rename the uwsgi file containing secrets from `uwsgi_db.ini` to something else now that there is a secret that is _not_ related to the databsae.

Reply:
[this PR](https://github.com/fedspendingtransparency/data-act-build-tools/pull/269) will make sure the API key is added. Once that is merged in we should be good to go there.